### PR TITLE
WX17 blue

### PR DESCRIPTION
**Pull request by webxoss**
- [x] WX17-020
- [x] WX17-071
- [x] WX17-062
- [x] WX17-065
- [x] WX17-002
- [x] WX17-016
- [x] WX17-017
- [x] WX17-018
- [x] WX17-030
- [x] WX17-041
- [x] WX17-042
- [x] WX17-043
- [x] WX17-067
- [x] WX17-069
- [x] WX17-070
- [x] WX17-019
- [x] WX17-029
- [x] WX17-031
- [x] WX17-044
- [x] WX17-045
- [x] WX17-063
- [x] WX17-064
- [x] WX17-066
- [x] WX17-068

### CODE REVIEW

File: CardInfo.js [hunk 1]
```diff
@@ -123960,15 +123960,110 @@ var CardInfo = {
...
-      "①あなたのトラッシュから無色のカード１枚を手札に加える。",
+    ],
+    spellEffect: {
+      getTargets: function () {
+        return this.player.opponent.signis.filter(function (signi) {
+          return signi.hasClass('精元');
+        },this);
+      },
+      actionAsyn: function (target) {
+        return this.game.bounceCardsToDeckAsyn([targets]);
```
Comment (Deardrops) [inline CardInfo.js]: typo: `target`

File: CardInfo.js [hunk 2]
```diff
@@ -124469,15 +124622,85 @@ var CardInfo = {
...
+          this.game.trashCards(cards);
+        });
+      },
+      actionAsyn: function () {
+        return this.banishSigniAsyn();
+      },
+    },{
+      costCoin: 2,
+      actionAsyn: function () {
+        return this.player.showCardsAsyn(this.player.opponent.hands).callback(this,function () {
```
Comment (Deardrops) [inline CardInfo.js]: 先处理对手手牌为0的情况？

File: CardInfo.js [hunk 3]
```diff
@@ -128130,13 +128357,24 @@ var CardInfo = {
...
+      "あなたがベットする場合、このアーツを使用するためのコストは《青×0》になる。\n" +
       "スペル１つの効果を打ち消す。"
     ],
-    "multiEner": false,
-    cardText: "何所までも深い、黒い闇……！～ピルルク～"
+    bet: 1,
+    bettedCost: {},
+    artsEffect: {
+      actionAsyn: function () {
+        return true;
```
Comment (Deardrops) [inline CardInfo.js:128375]: 唔，这里哪一句是取消一个效果？

**Review by Deardrops — APPROVED** [review]

File: CardInfo.js (hunk 3 above)
Comment (webxoss) [reply CardInfo.js:128375]: 当时的设计是返回值表示是否取消。
事实证明这样的设计很蠢。
（蠢团子w

**Pull request merged**